.github/workflows/pmem_test_matrix.yml

```diff
@@ -22,8 +22,8 @@ permissions: {}
 jobs:
   job:
     name: ${{ matrix.force_enable }}, ${{ matrix.test_script }}, ${{ matrix.os }}, ${{ matrix.build }}
-    if: github.repository == 'pmem/pmdk'
```

Comment:
${{ github.event.repository.fork == false }}

Reply:
Since for the `workflow_call` event this payload is the same as for the caller workflow and the `pmem_test.yml` may be triggered on a schedule event, `github.event.repository` is not available. Sorry, can't use it.

Reply:
.